src/css/your-order.css

```diff
@@ -0,0 +1,267 @@
+.your-order {
+  background-color:#7a3145;
+}
+
+.your-order-title {
+  font-weight: 600;
+  font-size: 48px;
+  line-height: 1;
+  letter-spacing: -0.01em;
+  text-transform: uppercase;
+  color: #fbfbfb;
+  margin-bottom: 29px;
+}
+.your-order-title-span {
+  color: rgba(251, 251, 251, 0.3);
+}
+
+.your-order-description {
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 1.2;
+  color: #fbfbfb;
+  margin-bottom: 40px;
+}
+
+.your-order-elements {
+display: flex;
+flex-direction: column;
+gap: 20px;
+width: 100%;
+}
+
+.your-order-form {
+  background: #fbfbfb;
+  border-radius: 30px;
+  padding: 40px 20px;
+  width: 100%;
+}
+
+.form-title {
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 1;
+  letter-spacing: -0.01em;
+  text-transform: uppercase;
+  color: #141414;
+  margin-bottom: 40px;
+}
+
+.span-from {
+  color: #7a3145;
+}
+
+.span-form {
+  color: #576f35;
+}
+
+.your-order-input {
+  border: 1px solid rgba(20, 20, 20, 0.2);
+  border-radius: 30px;
+  padding-top: 14px;
+  padding-left: 16px;
+  padding-bottom: 14px;
+  width: 100%;
+  height: 46px;
+  margin-bottom: 15px;
+}
+
+.your-order-input:hover,
+.your-order-input:focus {
+  border-color: #7a3145;
+}
+
+.your-order-input:not(:placeholder-shown):invalid {
+  border-color: #e74a3b;
+}
+
+.your-order-input:not(:placeholder-shown):valid {
+  border-color: #3cbc81;
+}
+
+.user-comment {
+  border-radius: 15px;
+  height: 91px;
+  padding-bottom: auto;
+  margin-bottom: 20px;
+  resize: none;
+}
+
+.your-order-input::placeholder,
+.user-comment::placeholder {
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 1.2;
+  letter-spacing: -0.02em;
+  color: #141414;
+}
+
```

Comment:
add here: 
.your-order-input:focus-within::placeholder {
  opacity: 0;
}
So the placeholder will disappear on focus